kerberos/hosts.yml

```diff
@@ -0,0 +1,59 @@
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_ssh_user: ec2-user
+    ansible_become: true
+    realm: confluent.example.com 
+    kdc_hostname: <hostname of machine with KDC installed>
+    admin_hostname: <hostname of machine with KDC installed>
+    keytab_source_dir: <path to your keytabs>
```

Comment:
did we document that this path has to end in a /
ie /path/to/keytabs/

Reply:
Good catch.  I have updated the hosts file details with additional notes/examples.